docs/src/prototypes/CopyToClipboard/index.tsx

```diff
@@ -27,27 +27,37 @@ const CopyToClipboardPrototype: React.FC<CopyToClipboardPrototypeProps> = props
 }
 
 const CopyToClipboardInMenu: React.FC = props => {
+  const targetRef = React.useRef<HTMLElement>(null)
```

Comment:
https://github.com/stardust-ui/react/pull/1900/files#r322141672

```suggestion
  const [target, setTarget] = React.useState<HTMLElement>(null)
```

Reply:
Oops, was not really applied :(